.vale/fixtures/RedHat/CaseSensitiveTerms/testvalid.adoc

```diff
@@ -20,6 +21,7 @@ Exec-Shield
 Exif
 FAQ
 Fedora™ Project
+foobar
```

Comment:
ISG:
fubar noun (also "foobar" and derivatives)
Do not use. This term is an acronym of a profanity that is sometimes used by developers in code. If you see this term in sample code, revise the example or do not publish it. Rewrite any text to omit the term.

Reply:
@rolfe - Thanks for catching this. I searched IBM style for the string 'foobar', which doesn't come up so I ploughed on. I see it now under fubar and foo.

Reply:
Fixed.